src/pineko/theory.py

```diff
@@ -192,7 +201,7 @@ def iterate(self, f, **kwargs):
                 f(name, grid, **kwargs)
             rich.print()
 
-    def opcard(self, name, grid):
+    def opcard(self, name, grid, tcard):
```

Comment:
here we only need `xif` from the theory card, right? Then it is maybe better to pass only `xif` to `opcard` instead of the whole card